application.md

```diff
@@ -1,5 +1,141 @@
+contributions to their respective project, and be active contributors to date.
+These are therefore 'core' developers that understand the code base very well.
+They must also be in a position to dedicate enough time to properly mentor GSoC
+students.
+A number of the mentors have been GSoC mentors in all the editions since 2013.
+How will you keep mentors engaged with their students? Each sub-organisation has
+weekly teleconferences to discuss theIr development, in these calls they will
+dedicate a time for the GSoC mentors and students to update the progress and
+problems encountered.
+
+/is that engaging enough? A Party or chocolates will have more effect,
+something that will make others to want to mentor in the future/
+
+## How will you help your students stay on schedule to complete their projects?
+
+The students are asked to keep a blog up to date with their progress.
+Additionally, from the timeline in their application and with the help of the
+mentors a number of milestones will be set.
+All this information will be publicly available.
+In case of problems or delays most of the mentors have experience in help the
+students to identify the causes to drag them behind.
+The organisation admins will keep a close eye on the projects with inexperienced
+mentors to be able to help before to late.
+If after all the efforts the student don't show interest or disappears, the
+student will be failed.
+
+## How will you get your students involved in your community during GSoC?
+
+Working together with the community is a very important aspect for all of the
+sub-organisations under OpenAstronomy.
+We have created a page with guidelines [1] on what students can do to increase
+their chances to be accepted into the OpenAstronomy GSoC programme.
+To ensure students interact with our communities before their applications are
+submitted, one of the requirements of applications will be that students must
+submit at least one patch as part of their application, which will necessarily
+require that the students interact with the existing developer community.
+Once we select students, we will encourage them to further discuss their project
+ideas with the community before the coding period starts in earnest.
+
+Each sub-organisation provides a number of communication channel to its users
+and developers.
+These include weekly Google hangouts, email lists, as well as an IRC chat room.
+We will expect the students to make use of all of these channels during their
+projects.
+We also will make specific requirements for the students which include weekly
+meetings with their mentors as well as weekly blog posts on the OpenAstronomy
+site describing their progress.
+They will also be invited to participate in the creation of posters,
+presentations and/or tutorials to show in upcoming scientific conferences.
+Students (as all contributors) will be required to write up a user guide and
+full documentation for their contribution.
```

Comment:
"for their contribution." -> ""for their contribution to be considered successful."